sample/src/App.tsx

```diff
@@ -51,6 +52,15 @@ Sentry.init({
         return context;
       },
     }),
+    new Integrations.HttpClient({
+      // This array can contain tuples of `[begin, end]` (both inclusive),
+      // Single status codes, or a combinations of both.
+      // default: [[500, 599]]
+      failedRequestStatusCodes: [[400, 405], [500, 505], 507],
```

Comment:
```suggestion
      failedRequestStatusCodes: [[400, 599]],
```

So more inclusive with errors?